src/vstart.sh

```diff
@@ -1130,6 +1134,16 @@ EOF
     fi
 }
 
+start_exporter() {
+    echo "Starting exporter daemon..."
```

Comment:
It's technically feasible to launch multiple ceph-exporters, but for a (single-host) vstart-deployed cluster I don't think it makes sense as they all would scrape the same daemons. If we ever manage to make vstart + cephadm work together, some type of network isolation (e.g.: kernel namespaces) would allow for multi ceph-exporter deployment. In the meantime... 😮‍💨

Reply:
Clarify: "exporter" could mean `ceph_exporter` or `node_exporter`

Reply:
@epuertat Thank you, Ernesto! You're right—multiple 'ceph-exporter' instances on a single-host 'vstart' setup would be redundant since they scrape the same metrics. Integrating 'cephadm`'in the future could address this by enabling network isolation for multi-exporter setups, but it adds complexity to 'vstart'. For now, I suggest sticking to a single exporter instance in 'vstart' while considering 'cephadm' integration as a future enhancement. Let me know your thoughts!

Reply:
@anthonyeleven Thanks for the suggested change!